Source/SwiftLintFramework/Models/Location.swift

```diff
@@ -26,7 +26,18 @@ public struct Location: CustomStringConvertible, Comparable {
         self.character = character
     }
 
-    public init(file: File, offset: Int) {
+    public init(file: File, byteOffset offset: Int) {
+        self.file = file.path
+        if let lineAndCharacter = file.contents.lineAndCharacterForByteOffset(offset) {
+            line = lineAndCharacter.line
+            character = lineAndCharacter.character
```

Comment:
I found that
```
"👨‍👩‍👧‍👧".characters.count // 4
```
and above example is very edge case and Xcode fails to position the column indicator sometimes.
<img width="815" alt="screenshot 2015-12-25 13 26 39" src="https://cloud.githubusercontent.com/assets/33430/12000386/f1466d3c-ab18-11e5-9e7d-5412fccdcb0a.png">

By fixing https://github.com/jpsim/SourceKitten/pull/129:
<img width="811" alt="screenshot 2015-12-25 15 18 15" src="https://cloud.githubusercontent.com/assets/33430/12000492/1b1b3956-ab1b-11e5-8c4d-59cb6c5fc1f2.png">